napari/utils/data.py

```diff
@@ -0,0 +1,17 @@
+"""
+This is proxy file for examples to get data from scikit image
+"""
+
+try:
```

Comment:
I'll let @jni make the final call on this, but I think we don't want to trigger a pip install under the hood without someone explicitly doing it.

I know I had voiced concerns about the examples not being run-able after this PR merges, but maybe the best thing to do is just inside our tutorials make it very clear that to run our examples you'll also need to install scikit-image and give instructions on how to do that. If we do that then I'd be ok dropping this function and leave the examples unchanged.

@jni your opinion?

Reply:
i agree, I don't mind a helpful message that says "scikit-image not found" (though perhaps the standard ModuleNotFound error is self-explanatory enough) ... but I don't think we should alter the users environment without requesting permission.

Reply:
I agree with this point of view, but my english is not enough fluent to suggest this documentation text.

Reply:
100%, no auto-installs. I think it's totally fine for the examples to only run with the libraries specified in `requirements/docs.txt`, and as Talley suggests, I think ModuleNotFound is indeed explanatory enough.

Reply:
Hm. I see that examples are not included to `napari` package. So maybe other option is to copy eg. astronaut to examples dir and switch to imageio?

Reply:
No, I would like to keep the repo size small, and in the future we might switch away from astronaut anyway.